src/runner/PlutoRunner.jl

```diff
@@ -1516,7 +1516,13 @@ const integrations = Integration[
 
             function table_data(x::Any, io::Context)
                 rows = Tables.rows(x)
-
+                if isempty(rows)
+                    return Dict{Symbol,Any}(
+                        :objectid => objectid2str(x),
+                        :schema => [],
+                        :rows => [],
+                    )
+                end
```

Comment:
We don't need to add this, right? If the table is empty, it should already return this dict.